pkg/supply/services/services.go

```diff
@@ -60,10 +60,8 @@ type IASCredentials struct {
 
 type UnifiedIdentityCredentials struct {
 	IASCredentials
-	AuthzURL         string                  `json:"authorization_url" validate:"required"`
-	AuthzBundleURL   string                  `json:"authorization_bundle_url" validate:"required_without=AuthzObjectStore"`
-	AuthzObjectStore *ObjectStoreCredentials `json:"authorization_object_store" validate:"required_without=AuthzBundleURL"`
-	AuthzInstanceID  string                  `json:"authorization_instance_id" validate:"required"`
+	AuthzURL        string `json:"authorization_url" validate:"required"`
```

Comment:
Can't we completely omit the authorization_url? Instead we just use the tenant url of the identity broker:
<img width="430" alt="image" src="https://user-images.githubusercontent.com/42929142/155299035-ee6374cb-3e16-4dde-a434-abf0394f51db.png">